<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>741</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Login_page">
     <widget class="QLineEdit" name="password_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>246</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>252</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Sans</family>
        <pointsize>14</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="general_info">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>241</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Please Login to continue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Sans</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>326</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>186</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="error_label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>410</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="converter_page">
     <widget class="QLabel" name="title_label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>171</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Currency Converter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="from_currency_box">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>86</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="to_currency_box">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>86</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="from_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>115</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="to_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>174</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="currency_input">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>410</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>410</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
